examples/social_network/main.cpp

```diff
@@ -0,0 +1,35 @@
+
+#include <iostream>
+#include <graaflib/algorithm/social_network.h>
+
+
+int main() {
+    // Create a graph to represent the social network
+    Graaf::Graph socialNetwork;

```

Comment:
```suggestion
    graaf::undirected_graph<V, E> graph{};
```
You need to specify the graph type here. I think `undirected` is reasonable for social networks. The `V` and `E` should be replaced with more reasonable types according to the specific use case